### ISSUE TITLE
Trust Doman vs trust domain

### ISSUE THREAD

**Issue by PieterKas** [oauth-wg/oauth-transaction-tokens#145]
I see different capitalisation of trust domain (sometimes Trust Domain). We define "Trust Domain", but we also use "trust domain" throughout the document. Should we be consistent with capitalisation? If not, what are the rules for being inconsistent?

**Comment by gffletch** [comment]
Yes, we should be consistent:) It seems like we'd only capitalize it if it's a proper noun? I'm very bad at English grammar rules :)

**Comment by tulshi** [comment]
I am of the opinion that anything we have defined within the Terminology section of the spec should be capitalized, so that it is clear that we are referring to that definition.

**Comment by PieterKas** [comment]
I like that - it avoids ambiguity or requiring readers to be sufficiently immersed in the English language to distinguish between proper nouns etc.
